apps/keycloak-franceconnect/README.md

```diff
@@ -0,0 +1,15 @@
+# Keycloak-FranceConnect Demo app
+
+This chart needs an existing secret in the namespace containing the clientSecret provided by FranceConnect. To create it, you can use the following command:
+
+```sh
+echo -n '<clientsecret>' > test-fc_clientsecret && \
+kubectl create secret generic clientsecret --from-file=./test-fc_clientsecret
+```
+
+This chart also needs a configmap `test-fc-realm` with a test-fc.json file pointing to the content of `/realm/test-fc.json` (don't forget to change the value of ClientId for the FranceConnect Provider).
```

Comment:
Something we could test is creating a `templates` (standard `Helm` convention) folder and adding a `ConfigMap.yaml` to it.    
This should create it at installation.  Do you have time to test this behaviour ? If not I may be able to test it next week.

Reply:
done, seems to work well :)